exif2timestream.py

```diff
@@ -581,6 +689,11 @@ def main(opts):
     # beginneth the actual main loop
     start_time = time()
     cameras = parse_camera_config_csv(opts["-c"])
+    try:
+        global EXIF_DATE_MASK
+        EXIF_DATE_MASK = opts["-m"]
```

Comment:
Nope you were right. i was getting errors because i didn't add -m to the other test cases. thanks!